orka-ipsw-setup.sh

```diff
@@ -0,0 +1,245 @@
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+error() {
+    echo -e "${RED}[ERROR]${NC} $1" >&2
+    exit 1
+}
+
+# Check if running as root
+if [[ $EUID -eq 0 ]]; then
+    error "This script should not be run as root. Please run as a regular user with sudo privileges."
+fi
+
+# Check if sudo is available
+if ! command -v sudo &> /dev/null; then
+    error "sudo is required but not available"
+fi
+
+log "Starting MacOS Orka VM setup..."
+
+# Enable Screen Sharing
+enable_screen_sharing() {
+    log "Enabling Screen Sharing..."
+    
+    # Enable Screen Sharing via System Preferences
+    sudo /System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart \
+        -activate -configure -access -on -clientopts -setvnclegacy -vnclegacy yes \
+        -clientopts -setvncpw -vncpw admin -restart -agent -privs -all
+    
+    log "Screen Sharing enabled"
+}
+
+# Enable Remote Login (SSH)
+enable_remote_login() {
+    log "Enabling Remote Login (SSH)..."
+        
+        # Add to admin group
+        sudo dscl . -append /Groups/admin GroupMembership admin
+        sudo dscl . -append /Groups/wheel GroupMembership admin
+        
+        # Create home directory
+        sudo createhomedir -c -u admin
+    
+    # Enable SSH
+    sudo systemsetup -setremotelogin on
+    
+    # Enable full disk access for remote users
+    sudo dseditgroup -o edit -a everyone -t group com.apple.access_ssh
+    
+    # Ensure admin user can SSH
+    sudo dseditgroup -o edit -a admin -t user com.apple.access_ssh
+    
+    # Start SSH service
+    sudo launchctl load -w /System/Library/LaunchDaemons/ssh.plist 2>/dev/null || true
+    
+    log "Remote Login enabled with full disk access for admin user"
+}
+
+# Configure macOS updates (Set to download-only for Tahoe)
+configure_macos_updates() {
+    log "Configuring macOS updates for Download Only..."
+    
+    # Set automatic update check to true, but disable automatic installation
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticCheckEnabled -bool true
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticDownload -bool true
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticallyInstallMacOSUpdates -bool false
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist CriticalUpdateInstall -bool false
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist ConfigDataInstall -bool false
+    
+    # Disable automatic installation of app updates from App Store
+    sudo defaults write /Library/Preferences/com.apple.commerce.plist AutoUpdate -bool false
+    sudo defaults write /Library/Preferences/com.apple.commerce.plist AutoUpdateRestartRequired -bool false
+    
+    log "macOS updates configured for Download Only mode"
+}
+
+# Disable FileVault (When running on Tahoe)
+disable_filevault() {
```

Comment:
Did we decide to keep the warning for FileVault?

Btw - there is no logic here specific to Tahoe - this warning would show no matter what the guest OS version is. 

But again - do we need this?

Reply:
will delete